test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/YearTest.scala

```diff
@@ -0,0 +1,329 @@
+    assertEquals(0L, lastBCYear.until(Year.of(10), CENTURIES))
+    assertEquals(0L, firstACYear.until(Year.of(11), CENTURIES))
+    assertEquals(0L, lastBCYear.until(Year.of(10), MILLENNIA))
+    assertEquals(0L, firstACYear.until(Year.of(11), MILLENNIA))
+
+    assertEquals(100L, lastBCYear.until(Year.of(100), YEARS))
+    assertEquals(100L, firstACYear.until(Year.of(101), YEARS))
+    assertEquals(10L, lastBCYear.until(Year.of(100), DECADES))
+    assertEquals(10L, firstACYear.until(Year.of(101), DECADES))
+    assertEquals(1L, lastBCYear.until(Year.of(100), CENTURIES))
+    assertEquals(1L, firstACYear.until(Year.of(101), CENTURIES))
+    assertEquals(0L, lastBCYear.until(Year.of(100), MILLENNIA))
+    assertEquals(0L, firstACYear.until(Year.of(101), MILLENNIA))
+
+    assertEquals(1000L, lastBCYear.until(Year.of(1000), YEARS))
+    assertEquals(1000L, firstACYear.until(Year.of(1001), YEARS))
+    assertEquals(100L, lastBCYear.until(Year.of(1000), DECADES))
+    assertEquals(100L, firstACYear.until(Year.of(1001), DECADES))
+    assertEquals(10L, lastBCYear.until(Year.of(1000), CENTURIES))
+    assertEquals(10L, firstACYear.until(Year.of(1001), CENTURIES))
+    assertEquals(1L, lastBCYear.until(Year.of(1000), MILLENNIA))
+    assertEquals(1L, firstACYear.until(Year.of(1001), MILLENNIA))
+
+    assertEquals(1L, lastBCYear.until(firstACYear, ERAS))
+    assertEquals(0L, firstACYear.until(Year.of(100), ERAS))
+    assertEquals(0L, lastBCYear.until(Year.of(-100), ERAS))
+  }
+
+  @Test def atDay(): Unit = {
+    for (t <- samples) {
+      assertEquals(LocalDate.of(t.getValue, 1, 1), t.atDay(1))
+      assertEquals(LocalDate.of(t.getValue, 12, 31), t.atDay(t.length()))
+    }
+
+    assertEquals(LocalDate.of(leapYear.getValue, 2, 29), leapYear.atDay(31 + 29))
+    assertEquals(LocalDate.of(leapYear.getValue, 3, 1), leapYear.atDay(31 + 30))
+
+    assertEquals(LocalDate.of(nonLeapYear.getValue, 3, 1), nonLeapYear.atDay(31 + 29))
+  }
+
+  @Test def isLeap(): Unit = {
+    assertTrue(leapYear.isLeap)
+    assertFalse(nonLeapYear.isLeap)
+
+    assertTrue(Year.isLeap(leapYear.getValue))
+    assertFalse(Year.isLeap(nonLeapYear.getValue))
+  }
+
+  @Test def isValidMonthDay(): Unit = {
+    val leapMonthDay = MonthDay.of(2, 29)
+    assertTrue(leapYear.isValidMonthDay(leapMonthDay))
+    assertFalse(nonLeapYear.isValidMonthDay(leapMonthDay))
+
+    val otherMonthDays = Month.values().map(m => MonthDay.of(m, m.minLength()))
+    for {
+      t <- samples
+      monthDay <- otherMonthDays
+    } {
+      assertTrue(t.isValidMonthDay(monthDay))
+    }
+  }
+
+  @Test def atMonthDay(): Unit = {
+    val leapMonthDay = MonthDay.of(2, 29)
+    val leapDate = LocalDate.of(leapYear.getValue(),
+        leapMonthDay.getMonthValue, leapMonthDay.getDayOfMonth)
+    assertEquals(leapDate, leapYear.atMonthDay(leapMonthDay))
+
+    val nonLeapDate = LocalDate.of(nonLeapYear.getValue(),
+      leapMonthDay.getMonthValue, 28)
```

Comment:
Shouldn't be 4 spaces identantion when breaking up a line in which several parameters are passed to a function (continuation line)?